### PR TITLE
docs: improving spanish translations

### PR DESCRIPTION
**Pull request by jotagarciaz**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by raisedadead** [comment]
See: https://github.com/freeCodeCamp/freeCodeCamp/pull/37525#issuecomment-548345753

**Review by jotagarciaz — COMMENTED** [review]
@claretgb please may you review this changes?

File: docs/i18n-languages/spanish/CONTRIBUTING.md [hunk 1]
```diff
@@ -23,14 +23,14 @@ Puedes optar por contribuir a cualquier área de tu interés:
 
 2. Ayuda a los campistas en nuestro [foro público](https://www.freecodecamp.org/forum/):
 
-    - ❓ [Aportando una retroalimentación](https://www.freecodecamp.org/forum/?max_posts=1).
-    - 💬 [Aportando comentarios](https://www.freecodecamp.org/forum/c/project-feedback?max_posts=1) a sus proyectos.
+    - ❓ [Respondiendo sus cuestiones](https://www.freecodecamp.org/forum/?max_posts=1).
```
Comment (Narshe1412) [inline docs/i18n-languages/spanish/CONTRIBUTING.md]: Respondiendo a sus preguntas
(missing 'a', 'cuestion' is a false friend from 'question')

File: docs/i18n-languages/spanish/README.md [hunk 2]
```diff
@@ -29,7 +30,7 @@ Puedes hacer un pull de estos test desde [freeCodeCamp's CDN](https://cdn.freeco
 
 Una vez que hayas conseguido una certificación, siempre la tendrás. Siempre tendrás permitido agregar el link a ella desde tu LinkedIn o tu CV. Y cuando tus futuros empleadores o clientes hagan click en ese link, verán tu certificación verificada.
 
-La única excepción a esto es en caso en que descubramos una violación a nuestras [Políticas de Honestidad Académica](https://www.freecodecamp.org/academic-honesty). Cuando atrapamos a las personas sin ambigüedad plagiando (presentando el código o los proyectos de otras personas como si fueran suyos sin citación), hacemos lo que deben hacer todas las instituciones de aprendizaje rigurosas: revocamos sus certificaciones y prohibimos a esas personas.
+La única excepción a esto es en caso en que descubramos una violación a nuestras [Políticas de Honestidad Académica](https://www.freecodecamp.org/academic-honesty). Cuando atrapamos a las personas plagiando inequivocamente (presentando el código o los proyectos de otras personas como si fueran suyos sin citación), hacemos lo que deben hacer todas las instituciones de aprendizaje rigurosas: revocamos sus certificaciones y prohibimos el acceso a esas personas.
```
Comment (Narshe1412) [inline docs/i18n-languages/spanish/README.md]: Si descubrimos que una persona ha plagiado inequivocamente

File: docs/i18n-languages/spanish/README.md [hunk 3]
```diff
@@ -103,7 +104,7 @@ La única excepción a esto es en caso en que descubramos una violación a nuest
 
 Una vez que hayas obtenido las 6 de estas certificaciones,podrás reclamar tu certificación de Desarrollo Full Stack de freeCodeCamp.org. Esta última distinción significa que has completado aproximadamente 1,800 horas de programación con una amplia gama de herramientas de desarrollo web.
 
-#### Legacy Certifications
+#### Certificaciones Legacy
```
Comment (Narshe1412) [inline docs/i18n-languages/spanish/README.md]: Certificaciones Obsoletas (Legacy)

File: docs/i18n-languages/spanish/README.md [hunk 4]
```diff
@@ -113,31 +114,36 @@ Además contamos con 3 certificaciones legacy de nuestro plan de estudios del 20
...
-- Un [grupo de Facebook](https://www.facebook.com/groups/freeCodeCampEarth/permalink/428140994253892/) con más de 100,000 miembros alrededor del mundo.
+- [Noticias sobre desarrollo](https://www.freecodecamp.org/news) publicaciones, gratis, codigo abierto, sin publicidad donde podrás subir artículos de tu blog.
+
 
 ### [Unite a nuestra comunidad aquí](https://www.freecodecamp.org/signin).
 
 ### Informar bugs y problemas
 
-Si piensas que encontraste un bug, primero lee [How to Report a Bug](https://www.freecodecamp.org/forum/t/how-to-report-a-bug/19543) y sigue las instrucciones. Si estás seguro de que es un bug nuevo y has confirmado que afecta a otras personas, continúa y crea un ticket en GitHub. Asegúrate de incluir tanta información como sea posible para poder reproducir el bug.
+Si piensas que encontraste un bug, primero lee [How to Report a Bug](https://www.freecodecamp.org/forum/t/how-to-report-a-bug/19543) y sigue las instrucciones.
```
Comment (Narshe1412) [inline docs/i18n-languages/spanish/README.md]: Si piensas que encontraste un error en el sitio (bug)

File: docs/i18n-languages/spanish/README.md [hunk 5]
```diff
@@ -113,31 +114,36 @@ Además contamos con 3 certificaciones legacy de nuestro plan de estudios del 20
...
-Si piensas que encontraste un bug, primero lee [How to Report a Bug](https://www.freecodecamp.org/forum/t/how-to-report-a-bug/19543) y sigue las instrucciones. Si estás seguro de que es un bug nuevo y has confirmado que afecta a otras personas, continúa y crea un ticket en GitHub. Asegúrate de incluir tanta información como sea posible para poder reproducir el bug.
+Si piensas que encontraste un bug, primero lee [How to Report a Bug](https://www.freecodecamp.org/forum/t/how-to-report-a-bug/19543) y sigue las instrucciones. 
+
+Si estás seguro de que es un bug nuevo y has confirmado que afecta a otras personas, continúa y crea un ticket en GitHub. Asegúrate de incluir tanta información como sea posible para poder reproducir el bug.
 
 ### Informar problemas de seguridad
 
-Por favor, no crees un ticket en GitHub sobre problemas de seguridad. En cambio, envía un correo electrónico a `security @ freecodecamp.org` y lo veremos de inmediato.
+Si crees haber encontrado una vulnerabulidad, por favor reportala responsablemente.No crees un ticket en GitHub para problemas de seguridad. En lugar de ello, envía un correo electrónico a `security @ freecodecamp.org` y lo veremos de inmediato.
```
Comment (Narshe1412) [inline docs/i18n-languages/spanish/README.md]: Si crees haber encontrado una vulnerabilidad avísanos de manera responsable, por favor.

**Review by Narshe1412 — CHANGES_REQUESTED** [review]
Except line 136 on the README.md file, other are minor suggestions and false friends. Please review

File: docs/i18n-languages/spanish/CONTRIBUTING.md (hunk 1 above)
Comment (jotagarciaz) [reply docs/i18n-languages/spanish/CONTRIBUTING.md]: > Respondiendo a sus preguntas
> (missing 'a', 'cuestion' is a false friend from 'question')

I don't understand this one, "cuestiones" is also use as "preguntas". I'm on it with the missing 'a'

File: docs/i18n-languages/spanish/README.md (hunk 2 above)
Comment (jotagarciaz) [reply docs/i18n-languages/spanish/README.md]: okay, you've got the point, I'm on it ;)

File: docs/i18n-languages/spanish/README.md [hunk 6]
```diff
@@ -126,15 +126,15 @@ Nuestra comunidad además cuenta con:
...
+Si piensas que encontraste un fallo, primero lee [How to Report a Bug](https://www.freecodecamp.org/forum/t/how-to-report-a-bug/19543) y sigue las instrucciones. 
 
-Si estás seguro de que es un bug nuevo y has confirmado que afecta a otras personas, continúa y crea un ticket en GitHub. Asegúrate de incluir tanta información como sea posible para poder reproducir el bug.
+Si estás seguro de que es un fallo nuevo y has confirmado que afecta a otras personas, continúa y crea un ticket en GitHub. Asegúrate de incluir tanta información como sea posible para poder reproducir el fallo.
 
 ### Informar problemas de seguridad
 
-Si crees haber encontrado una vulnerabulidad, por favor reportala responsablemente.No crees un ticket en GitHub para problemas de seguridad. En lugar de ello, envía un correo electrónico a `security @ freecodecamp.org` y lo veremos de inmediato.
+Si crees haber encontrado una vulnerabulidad, por favor, avisanos de manera responsable. No crees un ticket en GitHub para problemas de seguridad. En lugar de ello, envía un correo electrónico a `security @ freecodecamp.org` y lo veremos de inmediato.
```
Comment (Narshe1412) [inline docs/i18n-languages/spanish/README.md]: `encontrado una vulnerabulidad, por favor, ` 
encontrado una vulnerabilidad por favor,

First colon should be removed. vulnerability has a typo

File: docs/i18n-languages/spanish/CONTRIBUTING.md (hunk 1 above)
Comment (Narshe1412) [reply docs/i18n-languages/spanish/CONTRIBUTING.md]: `Cuestion` is usually translated from subject or matter. Or when in the 'asking' context, it's something you ask to learn the truth about that something. If you're asking with the purpose of learning about a topic, 'pregunta' is the correct term.
https://dle.rae.es/?id=TzZfwOK
https://dle.rae.es/?id=Bb1zrQ1

File: docs/i18n-languages/spanish/README.md (hunk 6 above)
Comment (claretgb) [reply docs/i18n-languages/spanish/README.md]: Respectfully, I don't think first colon should be removed. It is an added formula of courtesy, and as such should be properly separated from the sentence.

Comment (jotagarciaz) [reply docs/i18n-languages/spanish/README.md]: Since it's been 6 days I will proceed to mark this as a resolver conversation.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**